Ark.Tools.FtpClient.SftpClient/SftpClientConnection.cs

```diff
@@ -32,21 +32,6 @@ public class SftpClientConnection : FtpClientConnectionBase  //former KailFtpCli
         private const string _dsa = "1.2.840.10040.4.1";
         private const string _ecdsa = "1.2.840.10045.2.1";
 
-        [Obsolete("Use the constructor with URI", false)]
-        public SftpClientConnection(string host, NetworkCredential credentials, int port = 2222)
```

Comment:
Please 'notify' this to all internal devs. the sftp:// default port is 22, not 2222 so if the caller only has the Host and expect to use 2222, after this change it will be broken.

he has to create the Uri with new Uri("sftp://<host>:2222") and not new Uri("sftp://<host>") to have the same default behaviour.